### PR TITLE
Added documentation to classes

### PR DESCRIPTION
**Pull request by PurpleBooth**
For issue #5

### CODE REVIEW

File: pybuildkite/agents.py [hunk 1]
```diff
@@ -2,16 +2,26 @@
...
+        Construct the class
         """
         self.client = client
         self.path = base_url + 'organizations/{}/agents/'
 
     def list_all(self, organizations, name=None, hostname=None, version=None):
+        """
+        Returns a paginated list of an organization’s agents.
 
+        :param organizations: Organisation slug
```
Comment (pyasi) [inline pybuildkite/agents.py]: typo

File: pybuildkite/agents.py [hunk 2]
```diff
@@ -20,4 +30,11 @@ def list_all(self, organizations, name=None, hostname=None, version=None):
         return self.client.get(self.path.format(organizations), query_params)
 
     def get_agent(self, organization, agent_id):
+        """
+        Get an agent
+
+        :param organization: Organisation slug
```
Comment (pyasi) [inline pybuildkite/agents.py]: typo

File: pybuildkite/agents.py (hunk 1 above)
Comment (PurpleBooth) [reply pybuildkite/agents.py]: Oops my British spellings getting in the way!

**Comment by pyasi** [comment]
Ping me when you're ready for a final review, looks like you're still working

**Comment by PurpleBooth** [comment]
@pyasi Ready for review

File: pybuildkite/jobs.py [hunk 3]
```diff
@@ -2,17 +2,43 @@
...
     def get_job_log(self, organization, pipeline, build, job, get_text=False, get_html=False):
+        """
+        Get a job’s log output
+
+        :param organization: Organization slug
+        :param pipeline: Pipeline slug
+        :param build: Build number
+        :param job: Job id
+        :param get_text: Unused
+        :param get_html: Unused
```
Comment (pyasi) [inline pybuildkite/jobs.py]: Oooh good find we should fix this up. Not in this PR though =)

**Pull request merged**